packages/react-instantsearch/src/widgets/Snippet.js

```diff
@@ -15,29 +15,24 @@ import SnippetComponent from '../components/Snippet.js';
  * @propType {string} [tagName='em'] - the tag to be used for highlighted parts of the attribute
  * @example
  * import React from 'react';
- *
- * import { connectHits, Snippet, InstantSearch } from 'react-instantsearch/dom';
- *
- * const CustomHits = connectHits(({ hits }) =>
- * <div>
- *   {hits.map(hit =>
- *     <p key={hit.objectID}>
- *       <Snippet attributeName="description" hit={hit} />
- *     </p>
- *   )}
- * </div>
- * );
- *
+ * import { Snippet, InstantSearch, Hits } from "react-instantsearch/dom";
```

Comment:
Use single quote as the reset of the example.

Reply:
my prettier settings were on default